tox.ini

```diff
@@ -42,15 +58,7 @@ deps =
     online: pytest-rerunfailures
     online: pytest-timeout
     # Oldest Dependencies
-    oldestdeps: sunpy<4.0
-    oldestdeps: astropy<4.3
-    oldestdeps: gwcs<0.16
-    oldestdeps: numpy<1.18
-    oldestdeps: matplotlib<3.3
-    oldestdeps: mpl_animators<1.1
-    oldestdeps: reproject==0.7.1
-    oldestdeps: scipy<1.8.0
-    oldestdeps: dask<2021.5.0
+    oldestdeps: minimum_dependencies
     # Figure tests need a tightly controlled environment
     figure-!devdeps: matplotlib==3.7.2
     figure-!devdeps: astropy==5.3.3
```

Comment:
I can't comment on the line, but:

```python
extras =
    # Do not use `all` here because of https://github.com/spacetelescope/minimum_dependencies/issues/30
    plotting
    reproject
    tests